examples/integrated_gradients_transformers.ipynb

```diff
@@ -11,9 +11,9 @@
    "cell_type": "markdown",
```

Comment:
<p>some --&gt; for some</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/537/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>